doc/README.bash-completion

```diff
@@ -159,9 +159,8 @@ $ john --fo=
 
 For a jumbo build, the first 4 examples will not work, because the option name
 is ambiguous.
-A jumbo version also has the options --field-separator-char= and
---fix-state-delay=, so at least the first two letters of the option name
-must be specified.
+A jumbo version also has the options --field-separator-char=, so at least the
+first two letters of the option name must be specified.
```

Comment:
I decided to revert the changes to this file, because it mentions a specific Jumbo version for the examples.